rasa/utils/tensorflow/layers.py

```diff
@@ -143,263 +142,10 @@ def call(self, x: tf.Tensor) -> tf.Tensor:
-            np.arange(d_model)[np.newaxis, :],
-            d_model,
-        )
-
-        # apply sin to even indices in the array; 2i
-        angle_rads[:, 0::2] = np.sin(angle_rads[:, 0::2])
-
-        # apply cos to odd indices in the array; 2i+1
-        angle_rads[:, 1::2] = np.cos(angle_rads[:, 1::2])
-
-        pos_encoding = angle_rads[np.newaxis, ...]
-
-        return tf.cast(pos_encoding, dtype=tf.float32)
-
-    def __init__(
-        self,
-        num_layers: int,
-        d_model: int,
-        num_heads: int,
-        dff: int,
-        max_seq_length: int,
-        reg_lambda: float,
-        dropout_rate: float = 0.1,
-        attention_dropout_rate: float = 0.0,
-        unidirectional: bool = False,
-        name: Optional[Text] = None,
-    ) -> None:
-        super().__init__(name=name)
-
-        self.d_model = d_model
-        self.unidirectional = unidirectional
-
-        l2_regularizer = tf.keras.regularizers.l2(reg_lambda)
-        self._embedding = DenseWithSparseWeights(
-            units=d_model, kernel_regularizer=l2_regularizer
-        )
-
-        self._pos_encoding = self._positional_encoding(max_seq_length, self.d_model)
-
-        self._dropout = tf.keras.layers.Dropout(dropout_rate)
-
-        self._enc_layers = [
-            TransformerEncoderLayer(
-                d_model, num_heads, dff, dropout_rate, attention_dropout_rate
-            )
-            for _ in range(num_layers)
-        ]
-        self._layernorm = tf.keras.layers.LayerNormalization(epsilon=1e-6)
-
-    def call(self, x: tf.Tensor, pad_mask: tf.Tensor, training: tf.Tensor) -> tf.Tensor:
-
-        # adding embedding and position encoding.
-        x = self._embedding(x)  # (batch_size, seq_len, d_model)
-        x *= tf.math.sqrt(tf.cast(self.d_model, tf.float32))
-        x += self._pos_encoding[:, : tf.shape(x)[1], :] * (1 - pad_mask)
-        x = self._dropout(x, training=training)
-
-        pad_mask = tf.squeeze(pad_mask, -1)  # (batch_size, seq_len)
-        pad_mask = pad_mask[:, tf.newaxis, tf.newaxis, :]  # (batch_size, 1, 1, seq_len)
-        if self.unidirectional:
-            # add look ahead pad mask to emulate unidirectional behavior
-            pad_mask = tf.minimum(
-                1.0, pad_mask + self._look_ahead_pad_mask(tf.shape(pad_mask)[-1])
-            )  # (batch_size, 1, seq_len, seq_len)
-
-        for layer in self._enc_layers:
-            x = layer(x, pad_mask, training)  # (batch_size, seq_len, d_model)
-
-        # if normalization is done in encoding layers, then it should also be done
-        # on the output, since the output can grow very large, being the sum of
-        # a whole stack of unnormalized layer outputs.
-        return self._layernorm(x)  # (batch_size, seq_len, d_model)
-
-
 class InputMask(tf.keras.layers.Layer):
     def build(self, input_shape: tf.TensorShape) -> None:
-        initializer = tf.keras.initializers.GlorotUniform()
```

Comment:
Why did you remove the initializer?

Reply:
because `GlorotUniform` is a standard one

Reply:
I thought it is cleaner to use standard parameters